query.go

```diff
@@ -1108,8 +1122,7 @@ func (s *ChainService) sendTransaction(tx *wire.MsgTx, options ...QueryOption) e
 	// error as the reliable broadcaster will take care of broadcasting this
 	// transaction upon every block connected/disconnected.
 	if numReplied == 0 {
-		log.Debugf("No peers replied to inv message for transaction %v",
-			tx.TxHash())
+		log.Debugf("No peers replied to inv message for transaction %v", txHash)
```

Comment:
nit: leave as is to keep it wrapped at 80 chars

Reply:
re-wrapped, but both of these logs were under 80.  The cursor on this one says "81" at the end of this line, which is the limit.  No matter though.